eventdiscovery-sdk/src/main/res/layout/schedjoules_view_event_details_description.xml

```diff
@@ -7,7 +7,7 @@
             android:layout_height="wrap_content"
             android:layout_width="match_parent"
             android:paddingLeft="@dimen/schedjoules_activity_horizontal_margin"
```

Comment:
Oh, good catch! Yes, that was the whole point for that dimen, I accidentally changed it back in one of my last changes (when I changed it for the action and direction views, cause that don't need the left padding to change)
I've fixed it.